Automattic/pocket-casts-ios
# Onboarding: Adds the updated Welcome View design

| 📘 Project: #500 |
|:---:|

This adds the initial welcome view design update with confetti and support the Plus and New account states

## Screenshots

| iPhone SE 14.1 | iPhone 8  - 15.5 | iPhone 14 Pro Max - 16.1 |
|:---:|:---:|:---:|
|<img width="200" src="https://user-images.githubusercontent.com/793774/201459214-2789f655-3942-4d7d-a2c2-98d8673b8217.png">|<img width="200" src="https://user-images.githubusercontent.com/793774/201459239-d76ba0b9-2467-48ac-9968-2bd6e66bc5b0.png">|<img width="200" src="https://user-images.githubusercontent.com/793774/201459219-6a1a46f7-af74-4d5e-a865-4bfa61d9811a.png">|

<details>
  <summary>Video Demo: Plus Confetti</summary>

   https://user-images.githubusercontent.com/793774/201782074-a6e8329d-e474-409d-ba29-444ddea6cc86.mp4
</details>

<details>
  <summary>Video Demo: Normal Confetti</summary>

   https://user-images.githubusercontent.com/793774/201782069-b9167b20-c6ec-4e6f-a12c-78d9a3d94fdf.mp4
</details>

## To test

1. Launch the app
2. Tap on the Podcasts tab
3. Tap on the Create Folder button
4. Tap on the Login button
5. Tap on the Unlock button
6. ✅ Verify the Welcome view appears and [matches the designs](https://www.figma.com/file/PviUP0aiZctOprDuuYRwpb/Onboarding-%26-Upgrade-v2?node-id=406%3A26148&t=eKpw3p6ssfv1EmoO-4)
    1. The title says "Thank you"
    2. The Confetti is plus confetti
    3. The icon in the header is gold
8. ✅ Verify you see the plus confetti  
9. Tap the Done button
10. ✅ Verify the view is dismissed
11. In Xcode, open the PlusCoordinator
12. On Line 13, change `.plus` to `.newAccount`
13. Relaunch the app
14. Once you get to the view, ✅ Verify the view [matches the designs](https://www.figma.com/file/PviUP0aiZctOprDuuYRwpb/Onboarding-%26-Upgrade-v2?node-id=406%3A25546&t=eKpw3p6ssfv1EmoO-4)
    1. The title says "Welcome..."
    2. The Confetti is normal confetti
    3. The icon in the header is not gold
16. Tap the Find my Next Podcast button
17. ✅ Verify the view dismisses and you are brought to the Discover view

## Checklist
